cmake/thirdparty/SetupROCm.cmake

```diff
@@ -7,7 +7,7 @@
 # ROCM
```

Comment:
Is this file still needed?

Reply:
I believe so, for the ENABLE_HCC option. Are you proposing this file should be renamed as well?

Reply:
I think renaming this would be good.